components/Card/TCardEvent.vue

```diff
@@ -4,10 +4,10 @@
   </div>
   <div v-else class="flex space-x-4 border p-4">
     <div class="flex-grow">
-      <router-link
+      <NuxtLink
         :to="`/events/${node.id}`"
         class="font-bold leading-none underline hover:no-underline"
-        >{{ node.name }}</router-link
+        >{{ node.name }}</NuxtLink
```

Comment:
`>` is next line now.
I think it has in same line like: `</NuxtLink>`

Reply:
It's formatted automatically by prettier.